game/screen.hh

```diff
@@ -133,6 +133,7 @@ private:
 	std::unique_ptr<Dialog> m_dialog;
 };
 
+#ifndef _MSC_VER
```

Comment:
Why this?

Reply:
seems to be unsupported or buggy in MSVC. see also https://stackoverflow.com/questions/32321006/resolving-definitions-of-specialized-static-member-variables-of-templated-classe